### PR TITLE
GO 75 add documentation for models

### PR DESCRIPTION
**Pull request by vvuwei**
- Updated hasura version to 2.28.0.
- Updated version of go-lib to 0.2.37.
- Added doc-comments for MempoolOperation.
- Added doc-comments for ActivateAccount.
- Added doc-comments for Ballot.
- Added doc-comments for Delegation.
- Added doc-comments for DoubleBaking.
- Added doc-comments for DoubleEndorsing.
- Added doc-comments for DoublePreendorsing.
- Added doc-comments for DelegateDrain.
- Added doc-comments for Endorsement.
- Added doc-comments for GasStats.
- Added doc-comments for NonceRevelation.
- Added doc-comments for Origination.
- Added doc-comments for Proposal.
- Added doc-comments for RegisterGlobalConstant.
- Added doc-comments for Reveal.
- Added doc-comments for SetDepositsLimit.
- Added doc-comments for SmartRollupAddMessage.
- Added doc-comments for SmartRollupCement.
- Added doc-comments for SmartRollupExecute.
- Added doc-comments for SmartRollupOriginate.
- Added doc-comments for SmartRollupPublish.
- Added doc-comments for SmartRollupRecoverBond.
- Added doc-comments for SmartRollupRefute.
- Added doc-comments for SmartRollupTimeout.
- Added doc-comments for TransferTicket.
- Added doc-comments for TxRollupCommit.
- Added doc-comments for TxRollupReturnBond.
- Added doc-comments for TxRollupSubmitBatch.
- Added doc-comments for TxRollupRemoveCommitment.
- Added doc-comments for TxRollupRejection.
- Added doc-comments for TxRollupOrigination.
- Added doc-comments for TxRollupFinalizeCommitment.
- Added doc-comments for UpdateConsensusKey, Transaction, TxRollupDispatchTickets.

### CODE REVIEW

File: cmd/mempool/models/register_global_constant.go [hunk 1]
```diff
@@ -3,14 +3,14 @@ package models
...
-	tableName struct{} `pg:"register_global_constant"`
+	tableName struct{} `pg:"register_global_constant" comment:"register_constant operation - is used to register a global constant - Micheline expression that can be reused by multiple smart contracts."`
 	MempoolOperation
-	Source       string `json:"source"`
-	Fee          string `json:"fee"`
-	Counter      string `json:"counter"`
-	GasLimit     string `json:"gas_limit"`
-	StorageLimit string `json:"storage_limit"`
-	Value        JSONB  `json:"value"  pg:"value,type:jsonb"`
+	Source       string `json:"source"` // DISCUSS: This is sender address or constant address?
```
Comment (m-kus) [inline cmd/mempool/models/register_global_constant.go]: This is sender address (they will soon even rename it on the proto level)
Same for all manager operations (having source field)

File: cmd/mempool/models/register_global_constant.go [hunk 2]
```diff
@@ -3,14 +3,14 @@ package models
...
+	Fee          string `json:"fee"`    // DISCUSS
+	Counter      string `json:"counter" comment:"An account nonce which is used to prevent operation replay."`
+	GasLimit     string `json:"gas_limit" comment:"A cap on the amount of gas a given operation can consume."`
+	StorageLimit string `json:"storage_limit" comment:"A cap on the amount of storage a given operation can consume."`
+	Value        JSONB  `json:"value"  pg:"value,type:jsonb" comment:"Constant value."`
```
Comment (m-kus) [inline cmd/mempool/models/register_global_constant.go]: Micheline JSON format

File: cmd/mempool/models/sr_add_message.go [hunk 3]
```diff
@@ -6,12 +6,12 @@ type SmartRollupAddMessage struct {
...
-	GasLimit     int64    `json:"gas_limit,string"`
-	StorageLimit int64    `json:"storage_limit,string"`
-	Source       string   `json:"source,omitempty" index:"set_deposits_limit_source_idx"`
-	Message      []string `json:"message"`
+	Fee          int64    `json:"fee,string" comment:"Fee to the baker, produced block, in which the operation was included (micro tez)."` // DISCUSS
+	Counter      int64    `pg:",pk" json:"counter,string" comment:"An account nonce which is used to prevent operation replay."`
+	GasLimit     int64    `json:"gas_limit,string" comment:"A cap on the amount of gas a given operation can consume."`
+	StorageLimit int64    `json:"storage_limit,string" comment:"A cap on the amount of storage a given operation can consume."`
+	Source       string   `json:"source,omitempty" index:"set_deposits_limit_source_idx" comment:"Address of the account who has sent the operation."`
+	Message      []string `json:"message" comment:"Messages added to the smart rollup inbox."` // DISCUSS
```
Comment (m-kus) [inline cmd/mempool/models/sr_add_message.go]: Array of hex strings

File: cmd/mempool/models/sr_execute.go [hunk 4]
```diff
@@ -6,14 +6,14 @@ type SmartRollupExecute struct {
...
-	CementedCommitment string `json:"cemented_commitment"`
-	OutputProof        string `json:"output_proof"`
+	Fee                int64  `json:"fee,string" comment:"Fee to the baker, produced block, in which the operation was included (micro tez)."` // DISCUSS
+	Counter            int64  `pg:",pk" json:"counter,string" comment:"An account nonce which is used to prevent operation replay."`
+	GasLimit           int64  `json:"gas_limit,string" comment:"A cap on the amount of gas a given operation can consume."`
+	StorageLimit       int64  `json:"storage_limit,string" comment:"A cap on the amount of storage a given operation can consume."`
+	Source             string `json:"source,omitempty" index:"sr_execute_source_idx" comment:"Address of the account who has sent the operation."` // DISCUSS
+	Rollup             string `json:"rollup" index:"sr_execute_rollup_idx" comment:"Smart rollup to which the operation was sent."`
+	CementedCommitment string `json:"cemented_commitment" comment:"Executed commitment."`
+	OutputProof        string `json:"output_proof" comment:"Output proof bytes (in base64)."`
```
Comment (m-kus) [inline cmd/mempool/models/sr_execute.go]: Hex string, not base64

File: cmd/mempool/models/sr_originate.go [hunk 5]
```diff
@@ -6,15 +6,15 @@ type SmartRollupOriginate struct {
...
-	Kernel           string `json:"kernel"`
-	OriginationProof string `json:"origination_proof"`
-	ParametersTy     string `json:"parameters_ty"`
+	Fee              int64  `json:"fee,string" comment:"Fee to the baker, produced block, in which the operation was included (micro tez)."` // DISCUSS
+	Counter          int64  `pg:",pk" json:"counter,string" comment:"An account nonce which is used to prevent operation replay."`
+	GasLimit         int64  `json:"gas_limit,string" comment:"A cap on the amount of gas a given operation can consume."`
+	StorageLimit     int64  `json:"storage_limit,string" comment:"A cap on the amount of storage a given operation can consume."`
+	Source           string `json:"source,omitempty" index:"sr_originate_source_idx" comment:"Address of the account who has sent the operation."` // DISCUSS
+	PvmKind          string `json:"pvm_kind" comment:"PVM kind (arith or wasm)."`
+	Kernel           string `json:"kernel" comment:"Kernel bytes (in base64)."`
```
Comment (m-kus) [inline cmd/mempool/models/sr_originate.go]: Hex

File: cmd/mempool/models/sr_originate.go [hunk 6]
```diff
@@ -6,15 +6,15 @@ type SmartRollupOriginate struct {
...
+	OriginationProof string `json:"origination_proof" comment:"Origination proof bytes (in base64)."`
```
Comment (m-kus) [inline cmd/mempool/models/sr_originate.go]: Hex

File: cmd/mempool/models/sr_originate.go [hunk 7]
```diff
@@ -6,15 +6,15 @@ type SmartRollupOriginate struct {
...
+	ParametersTy     string `json:"parameters_ty" comment:"Smart rollup parameter type."`
```
Comment (m-kus) [inline cmd/mempool/models/sr_originate.go]: Micheline JSON

**Review by aopoltorzhicky — APPROVED** [review]

**Pull request merged**